plugins/services/src/js/containers/service-connection/ServiceConnectionEndpointList.js

```diff
@@ -214,4 +244,12 @@ ServiceConnectionEndpointList.contextTypes = {
   router: routerShape
 };
 
+function hasWebUI(service) {
```

Comment:
What is the point of this check? Can't we just make `getWebURL` return null or "'?